images/kubekins-e2e/e2e-runner.sh

```diff
@@ -20,7 +20,7 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+export PS4='+(${BASH_SOURCE[0]}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
 echo 'WARNING: e2e-runner.sh is deprecated! This will error on August 2nd' >&2
```

Comment:
@krzyzacy should we delete this script? (can be in a follow up)

Reply:
lol how come e2e-runner is still alive :-)

Reply:
same thing with the build-cop scripts below :man_shrugging: and the other e2e runner below